pages/docs/platform/monitor/insights.mdx

```diff
@@ -0,0 +1,209 @@
+import { Info } from "src/shared/Docs/mdx";
+
+export const description = 'Query and analyze event data with SQL in the Inngest platform.';
+
+# Insights
+
+Inngest Insights allows you to query and analyze your event data using SQL directly within the Inngest platform. Every event sent to Inngest contains valuable information, and Insights gives you the power to extract meaningful patterns and analytics from that data.
+
+<Info>
+  Insights support is currently in Public Beta. Some details including SQL syntax and feature availability are still subject to change during this period. Read more about the Public Beta [roadmap here](#roadmap).
+</Info>
+
+## Overview
+
+Insights provides an in-app SQL editor and query interface where you can:
+- Query event data using familiar SQL syntax
+- Save and reuse common queries
+- Analyze patterns in your event streams
+- Extract business intelligence from your workflows
+
+Currently, you can **only query events**. Support for querying function runs will be added in future releases.
+
+## Getting Started
+
+Access Insights through the Inngest dashboard by clicking on the "Insights" tab in the left navigation. 
+
+![Getting Started Dashboard View](/assets/docs/platform/monitor/insights/insights_dashboard_view.png)
+
+We have several pre-built query templates to help you get started exploring your data.
+
+![Getting Started Templates View](/assets/docs/platform/monitor/insights/insights_template_view.png)
+
+## SQL Editor
+
+The Insights interface includes a full-featured SQL editor where you can:
+
+- Write and execute SQL queries against your event data
+- Save frequently used queries for later access
+- View query results in an organized table format
+- Access query history and templates from the sidebar
+
+![Sql Editor View](/assets/docs/platform/monitor/insights/insights_sql_editor.png)
+
+### Available Columns
+
+When querying events, you have access to the following columns:
+
+| Column | Type | Description |
+|--------|------|-------------|
+| `event_id` | String | Unique identifier for the event |
+| `event_name` | String | The name/type of the event |
+| **`event_data`** | **JSON** | **The event payload data - users can send any JSON structure here** |
+| `event_ts` | DateTime | Timestamp when the event occurred |
+| `event_v` | String | Event format version |
+
+For more details on the event format, see the [Inngest Event Format documentation](https://www.inngest.com/docs/features/events-triggers/event-format).
+
+### Data Retention
+
+You can query events from **up to 30 days in the past**. Older events are not available through Insights.
+
+### Result Limits
+
+- Current page limit: **1000 rows**
+- Future updates will support larger result sets through async data exports
+- Pagination support is planned for future releases
```

Comment:
I would leave this point out, we might implement frontend pagination but I don't think that's worth mentioning